.circleci/config.yml

```diff
@@ -27,34 +341,93 @@ commands:
             docker build -t app:build -f << parameters.dockerfile >> --label git.commit="$CIRCLE_SHA1" .
             docker tag app:build "${DOCKERHUB_REPO}":<< parameters.image_tag >>
             docker push "${DOCKERHUB_REPO}":<< parameters.image_tag >>
+
 jobs:
-  build:
-    machine: true
-    working_directory: ~/addons-server
+  test:
+    <<: *defaults
+    parameters:
+      toxenv:
+        type: string
+    steps:
+      - checkout
+      - run:
+          name: Initial setup
+          command: |
+            sudo apt-get update
+            sudo apt-get install -y gettext pngcrush librsvg2-bin
+            sudo cp ./docker/etc/mime.types /etc/mime.types
+            sudo touch /addons-server-docker-container
+      - node/install:
+        node-version: 12.20.0
+      - run:
+          name: Install dockerize
+          command: |
+            wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+            sudo tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+            rm dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+          environment:
+            DOCKERIZE_VERSION: v0.6.1
+      - run:
+          name: Wait for redis
+          command: dockerize -wait tcp://localhost:6379 -timeout 1m
+      - run:
+          name: Wait for mysql
+          command: dockerize -wait tcp://localhost:3306 -timeout 1m
+      - run:
+          name: Wait for memcached
+          command: dockerize -wait tcp://localhost:11211 -timeout 1m
+      - run:
+          name: Wait for elasticsearch
+          command: dockerize -wait tcp://localhost:9200 -timeout 1m
+      - run: pip install tox
+      - run: pip install --no-deps -r requirements/ci_base.txt
+      - run:
+          command: tox -e << parameters.toxenv >>
+          environment:
+            RUNNING_IN_CI: True
```

Comment:
Do we have anything that still depends on this ? Maybe we can remove it.

Reply:
it's still referenced in our code:

```
rg RUNNING_IN_CI
src/olympia/amo/search.py
29:    if os.environ.get('RUNNING_IN_CI'):
```

Reply:
I think it should be useless - it's just to set up some fake auth ?

Reply:
lets see if removing it does anything